.github/workflows/validate_images.yml

```diff
@@ -0,0 +1,22 @@
+name: Validate Images
```

Comment:
I think this would fit better within the `Validate packs` workflow! That all the validations are just jobs in one workflow

Reply:
ye I was thinking that but didn't want to do it until I checked, have combined them both, let me know if that's right